src/app/pages/pages-routing.module.ts

```diff
@@ -1,9 +1,10 @@
-import { RouterModule, Routes } from '@angular/router';
-import { NgModule } from '@angular/core';
+import {RouterModule, Routes} from '@angular/router';
+import {NgModule} from '@angular/core';
 
-import { PagesComponent } from './pages.component';
-import { DashboardComponent } from './dashboard/dashboard.component';
-import { MachinesListComponent } from './machines/machines.list.component';
+import {PagesComponent} from './pages.component';
```

Comment:
Ale w zasadzie jak to jest, że mamy pozapinane lintery, a takie commity są ciągle?